app/src/components/v-form/form-field.vue

```diff
@@ -133,6 +133,7 @@ export default defineComponent({
 		const internalValue = computed(() => {
 			if (props.modelValue !== undefined) return props.modelValue;
 			if (props.initialValue !== undefined) return props.initialValue;
+			if (props.field.meta?.interface === 'input-rich-text-html') return '';
```

Comment:
We can't rely on hardcoding any exceptions based on the specific interface. Seeing that interfaces are also extension types, this name could be anything at any time. I think we'll have to look for the fix on the TinyMCE side itself!

Reply:
That does makes sense. Do you have a particular suggestion/proposal to TinyMCE vue? The immediate thought in my mind is turning `typeof val === 'string'` into  `typeof val === 'string' || val === null`, but it might not work since `setContent()` expects a string. Reference: https://www.tiny.cloud/docs/api/tinymce/tinymce.editor/#setcontent

or doing something like turning this

```js
if (editor && typeof val === 'string' && val !== prevVal && val !== editor.getContent({ format: props.outputFormat })) {
      editor.setContent(val);
}
```

into

```js
if (val === null) {
      editor.setContent('');
} else if (editor && typeof val === 'string' && val !== prevVal && val !== editor.getContent({ format: props.outputFormat })) {
      editor.setContent(val);
}
```

_hope I didn't make anyone's eyes bleed with my code above_ 😅

Reply:
Your last example makes sense to me! When `val` is explicit `null`, we'll have to empty out the editor by inserting an empty string. 

That being said, we'll have to make sure that that doesn't then stage an empty string back up to the parent, causing an infinite loop.

Reply:
Just tested it by editing the code in node_modules ( not sure if there's a better way D: ) and it works flawlessly. Though with an additional `editor` in the if statement, becoming `if (editor && val === null)` just to be safe as the original code, seems to make sense. Will create the PR over at TinyMCE Vue 👍

Reply:
Nice! I was actually under the impression that your code example above was part of Directus, not TinyMCE :o

Reply:
oh haha. Honestly I still tried to see is it possible to add the fix within Directus, but I think we'll then need to add a watcher to the value, check did it turn to null (which is after the watcher in tinymce finish firing), then set it to an empty string ourselves.

That'll be unfavorable right? or am I inadvertently overcomplicating things here? 🤔

Reply:
Did you confirm the same problem exists within tinymce standalone? I wouldn't want to bother them with a PR that's specifically addressing an issue that only shows up within Directus 🙂

Reply:
hmm currently the tinymce-vue's watcher does have the typeof string check, hence it doesn't clear it's content if we set it as null. Definitely share the same sentiment of trying not to bother them with a PR if possible as well haha. 

Alright just to confirm the current expectation, if we were able to make this fix occur within `input-rich-text-html.vue`, that is acceptable right? Previously since I added the fix in in `form-field.vue`, hence that is an "unstable" fix (which I wholeheartedly agree)?

Reply:
Oh absolutely, if it's within the interface that has the problem (in this case input-rich-text-html), then that's totally fine. As a matter of fact, that interface itself already does a lot of type checking and refreshing logic to make TinyMCE behave as it should, so it could very well be that this problem is actually caused by us missing a particular check somewhere 🙂

Reply:
ah got it. Once again I got confused with your previous comment:

> I think we'll have to look for the fix on the TinyMCE side itself!

and my first thought was the package itself, whereas you actually meant TinyMCE as in the interface... Oh my, sorry for the confusion again. 😂 Now I definitely agree that the check should be in the interface. Thank you for clarifying!

Reply:
Ohhhhh I see! Yeah haha I can see how that's confusing 👉🏻👈🏻 I meant that more as the "Side where _we_ use TinyMCE" 😁

Reply:
So looks like the issue can be fixed here:

https://github.com/directus/directus/blob/a6e88183e99847c96c956add94a148965b9e50be/app/src/interfaces/input-rich-text-html/input-rich-text-html.vue#L290-L297

where instead of `return props.value` (which is null, thus not triggering tinymce watcher), doing `return ''` works.

however I would still need to defer to you on whether `return ''` is enough or should we do `return props.value || ''` which also works. Just depends on whether falsy values will creep past the if statement above it 🤔

Reply:
How about we update it to:

```js
 		const internalValue = computed({ 
 			get() { 
 				if (!props.value) return '';  
 				return replaceTokens(props.value, getToken()); 
 			}, 
```

Reply:
oh true that's much better. It works perfectly! thank you very much for the prompt feedback

Reply:
Thanks (once again!) for all your help 🙂

Reply:
```suggestion
```

Reply:
shoot forgot to remove this change 🤦‍♂️ thanks haha